doc/source/gallery/pandas_accessor.ipynb

```diff
@@ -0,0 +1,141 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using GeoPandas via the pandas \"geo\" accessor\n",
+    "\n",
+    "GeoPandas can be used from pandas with the \"geo\" accessor, which is registered on `pandas.Series` when `geopandas` is first imported."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use this import to register the \"geo\" accessor.\n",
+    "import geopandas.accessors  # noqa # pylint: disable=unused-import\n",
+    "from geopandas.array import GeometryDtype\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from shapely import Point"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using the Series.geo accessor\n",
+    "\n",
+    "Construct a Series using the GeometryDtype."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    POINT (1 2)\n",
+       "1    POINT (3 4)\n",
+       "2    POINT (5 6)\n",
+       "dtype: geometry"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "s = pd.Series(\n",
+    "    [Point(1, 2), Point(3, 4), Point(5, 6)],\n",
+    "    dtype=GeometryDtype(),\n",
```

Comment:
```suggestion
    "    dtype="geometry",\n",
```
if geopandas has been imported "geometry" should work, and then the import from geopandas.array isn't needed.